### PR TITLE
Prevent form from closing if validation fails

### PR DESCRIPTION
**Pull request by davidjumani**

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/views/AutogenView.vue [hunk 1]
```diff
@@ -831,6 +831,11 @@ export default {
             this.actionLoading = true
             for (const params of paramsList) {
               api(this.currentAction.api, params).then(json => {
+                setTimeout(() => {
```
Comment (rhtyd) [inline src/views/AutogenView.vue:834]: Would this cause an issue if the form is closed for group actions?

**Comment by rhtyd** [comment]
@davidjumani general question - if we're not handling in `finally` Do we want the form to stay open in case of error?

**Comment by davidjumani** [comment]
@rhtyd Yes, in case there was a network or validation error. That way the user won't have to enter all the details all over again

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@davidjumani can you address merge conflict?
@utchoang can you help review/test this? Thanks

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Pull request merged**